liesel/model/model.py

```diff
@@ -758,12 +773,26 @@ def transform(
         0.0
         """
 
+        warnings.warn(
+            "GraphBuilder.transform is deprecated. Use Var.transform instead. "
+            "This method will be removed in v0.4.0",
+            FutureWarning,
+        )
+
         if var.weak:
             raise RuntimeError(f"{repr(var)} is weak")
 
         if var.dist_node is None:
             raise RuntimeError(f"{repr(var)} has no distribution")
 
+        if bijector is not None and not isinstance(bijector, type) and (args or kwargs):
```

Comment:
can we test here for `isintance(bijector, BijectorType)` instead of `bijector is not None and not isinstance(bijector, type)`?

would make it better comprehensible

Reply:
I don't know about a `BijectorType` object, but I clarified the check by introducing

```python
def is_bijector_class(obj):
    return isinstance(obj, type) and issubclass(obj, jb.Bijector)
```

and using

```python
if not is_bijector_class(bijector) and (args or kwargs):
```

now :)